compiler/rustc_metadata/src/rmeta/decoder/cstore_impl.rs

```diff
@@ -454,6 +454,11 @@ impl CStore {
         self.get_crate_data(def_id.krate).module_expansion(def_id.index, sess)
     }
 
+    /// Only public-facing way to traverse all the definitions in a non-local crate
+    pub fn num_def_ids_untracked(&self, cnum: CrateNum) -> usize {
```

Comment:
Without a comment explicitly telling that this function is used by a third-party project this function will be eventually removed by some cleanup.